distribution_metadata.c

```diff
@@ -584,6 +780,27 @@ LoadShardIntervalRow(int64 shardId, Oid *relationId, char **minValue,
 
 
 /*
+ * NextSequenceId allocates and returns a new unique id generated from the given
+ * sequence name.
+ */
+static uint64
+NextSequenceId(char *sequenceName)
```

Comment:
We don't pass it into any other functions, and in the previous implementation of this function (essentially unchanged here) it was just a `#define` we passed into `makeRangeVar` or whatnot, so I don't see why it needs to be exposed: this function is for getting sequence values related to metadata (see the previous implementation, which was locked to the metadata schema), so it should encapsulate the knowledge of where they live.